### PR TITLE
added classes controllers and jsp files what need to this classes

### PR DESCRIPTION
**Pull request by Fedich09**

### CODE REVIEW

File: src/main/java/mate/controllers/IndexController.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package mate.controller;
+package mate.controllers;
```
Comment (MykolaFatkullin) [inline src/main/java/mate/controllers/IndexController.java]: Don’t use the plural for package names

File: src/main/java/mate/controllers/cars/AddingDriverToCarController.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package mate.controllers.cars;
```
Comment (MykolaFatkullin) [inline src/main/java/mate/controllers/cars/AddingDriverToCarController.java]: Don’t use the plural for package names

File: src/main/java/mate/controllers/cars/AddingDriverToCarController.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        req.getRequestDispatcher("/WEB-INF/views/cars/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String carId = req.getParameter("carId");
+        String driverId = req.getParameter("driverId");
+        carService.addDriverToCar(driverService.get(Long.valueOf(driverId)),
+                carService.get(Long.valueOf(driverId)));
```
Comment (MykolaFatkullin) [inline src/main/java/mate/controllers/cars/AddingDriverToCarController.java]:
```suggestion
        carService.addDriverToCar(driverService.get(Long.valueOf(driverId)),
                carService.get(Long.valueOf(carId)));
```
I guess it must be like this =)

**Review by MykolaFatkullin — CHANGES_REQUESTED** [review]

File: src/main/java/mate/controllers/cars/AddingDriverToCarController.java (hunk 3 above)
Comment (mfedenko09) [reply src/main/java/mate/controllers/cars/AddingDriverToCarController.java]: yes, your right))

**Review by MykolaFatkullin — APPROVED** [review]

File: src/main/java/mate/controllers/CreateNewDriverController.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/driver/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String driverName = req.getParameter("driverName");
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/controllers/CreateNewDriverController.java]:
```suggestion
        String driverName = req.getParameter("driver_name");
```

File: src/main/java/mate/controllers/CreateNewDriverController.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        String licenseNumber = req.getParameter("licenseNumber");
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/controllers/CreateNewDriverController.java]:
```suggestion
        String licenseNumber = req.getParameter("license_number");
```

File: src/main/java/mate/controllers/DisplayAllDriversController.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+public class DisplayAllDriversController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Driver> listDrivers = driverService.getAll();
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/controllers/DisplayAllDriversController.java]:
```suggestion
        List<Driver> listOfDrivers = driverService.getAll();
```
or just drivers

File: src/main/java/mate/controllers/IndexController.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/mate/controllers/IndexController.java]: controller — singular, 
controllers — plural;
So the package name should be `controller`

File: src/main/java/mate/controllers/cars/AddingDriverToCarController.java (hunk 2 above)
Comment (dmytro-hryhoruk) [reply src/main/java/mate/controllers/cars/AddingDriverToCarController.java]: ?

File: src/main/java/mate/controllers/cars/AddingDriverToCarController.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddingDriverToCarController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/controllers/cars/AddingDriverToCarController.java]:
```suggestion
public class AddDriverToCarController extends HttpServlet {
```

File: src/main/webapp/WEB-INF/views/car/add.jsp [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+    <title>Add driver to car</title>
+</head>
+<body>
+<h1>Add drivers to car</h1>
+<h4 style="color: #ff0000">${messageCarIdNorExist}</h4>
+<h4 style="color: #ff0000">${messageDriverIdNotExist}</h4>
+<form method="post" action="${pageContext.request.contextPath}/car/add">
+    Input car id: <input type="number" min="1" name="carId">
+    Input driver id: <input type="number" min="1" name="driverId">
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/car/add.jsp]: let's use required attribute

File: src/main/webapp/web.xml [hunk 9]
```diff
@@ -5,11 +5,55 @@
...
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>createNewDriver</servlet-name>
+        <servlet-class>mate.controllers.CreateNewDriverController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>createNewDriver</servlet-name>
+        <url-pattern>/driver/create</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]: https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/08_servlets_practice_checklist.html 
check urls naming for each url

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/mate/dao/jdbc/CarJdbcImpl.java [hunk 10]
```diff
@@ -89,19 +109,35 @@ public Car create(Car car) {
...
+                + "    d.license_number "
+                + "FROM "
+                + "    cars AS c"
+                + "        RIGHT JOIN"
+                + "    manufacturers AS m ON c.manufacturer_id = m.manufacturer_id"
+                + "        RIGHT JOIN"
+                + "    cars_drivers AS cd ON c.car_id = cd.car_id"
+                + "        RIGHT JOIN"
+                + "    drivers AS d ON d.driver_id = cd.driver_id"
+                + "    where c.is_delete = false and m.is_delete = false ";
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/dao/jdbc/CarJdbcImpl.java:138]: Are you trying to fetch the whole DB with one query?

Comment (dmytro-hryhoruk) [reply src/main/java/mate/dao/jdbc/CarJdbcImpl.java:138]: discribe the purpose of this in your chat please

File: src/main/webapp/web.xml [hunk 11]
```diff
@@ -7,9 +7,53 @@
...
+        <servlet-name>displayAllDrivers</servlet-name>
+        <servlet-class>mate.controller.driver.DisplayAllDriversController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>displayAllDrivers</servlet-name>
+        <url-pattern>/drivers</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>createNewManufacturerController</servlet-name>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml:34]:
```suggestion
        <servlet-name>createNewManufacturer</servlet-name>
```

File: src/main/webapp/web.xml [hunk 12]
```diff
@@ -7,9 +7,53 @@
...
+        <url-pattern>/cars/create</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>addingDriverToCarController</servlet-name>
+        <servlet-class>mate.controller.car.AddDriverToCarController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>addingDriverToCarController</servlet-name>
+        <url-pattern>/cars/add</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml:57]:
```suggestion
        <url-pattern>/cars/drivers/add</url-pattern>
```

File: src/main/webapp/web.xml (hunk 11 above)
Comment (dmytro-hryhoruk) [reply src/main/webapp/web.xml:34]: remove this from servlets names

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**